Source/JavaWrappers.cpp

```diff
@@ -318,6 +318,31 @@ namespace java::io
     {
         return m_env->CallIntMethod(JObject(), m_env->GetMethodID(m_class, "read", "([B)I"), (jbyteArray)byteArray);
     }
+
+    OutputStream::OutputStream(jobject object)
+            : Object{object}
```

Comment:
nit: I don't know why Android Studio formats this with so many spaces.
```suggestion
        : Object{object}
```